scripts/build_dockerfiles.sh

```diff
@@ -19,7 +19,14 @@
 # Error out if any of the commands fail.
 set -e
 
-docker build -t gcr.io/blueshift-playground/blueshift:cpu -f- . <Dockerfile
+# GPU base-base, with all CUDA dependencies required for GPU work. Built off of the NVIDIA base image.
+docker build -t gcr.io/blueshift-playground/blueshift:gpu-base -f- . <dockerfiles/Dockerfile.gpu
```

Comment:
now we can build our own GPU base image that's slimmer.

Reply:
nice, that's a practical, every-day benefit for users